src/js/apps/globals/nav_app.js

```diff
@@ -154,22 +186,18 @@ export default App.extend({
   },
   onStart() {
     const currentUser = Radio.request('bootstrap', 'currentUser');
+    this.showAdminTools = currentUser.can('dashboards:view') || currentUser.can('clinicians:manage') || currentUser.can('programs:manage');
```

Comment:
If the user has none of those permissions, the `Admin Tools` droplist won't be shown.

![Screenshot 2023-01-12 at 10 01 54 AM](https://user-images.githubusercontent.com/35355575/212998359-5a102297-9d29-436e-ba77-ace502ac7c9a.png)

Thought about doing this instead:

```js
canDoAny(...props) {
  const role = this.getRole();
  const permissions = role.get('permissions');

  return props.some(prop => permissions.includes(prop));
}
```

And then you could do this instead:

```js
this.showAdminTools = currentUser.canDoAny('dashboards:view', 'clinicians:manage', 'programs:manage');
```

But figured that was getting too cute.